src/liboslexec/opclosure.cpp

```diff
@@ -86,7 +86,7 @@ osl_closure_to_ustringhash(ShaderGlobals* sg, ClosureColor* c)
     // Special case for printing closures
     std::ostringstream stream;
     stream.imbue(std::locale::classic());  // force C locale
-    print_closure(stream, c, &sg->context->shadingsys());
+    print_closure(stream, c, &sg->context->shadingsys(), true);
```

Comment:
add comment to the `true` parameter identifying the parameter name "treat_string_as_ustring_hash"?